src/Components/Bill/Bill.jsx

```diff
@@ -63,64 +60,47 @@ function Bill({ billData, updateBillData }) {
         <div className="button-wrapper">
           <input
             type="button"
-            className="button"
+            className={`button ${clickedButton === '5%' ? 'clicked' : ""}`} 
             name="tipPercent"
             value="5%"
-            onClick={setValueFromForm}
+            onClick={() => setValueFromForm('5%')}
```

Comment:
I like where we went with the tip percentage button today.  Good user empathy and a good adjustment to the onClick to pass off the value of the actual button that was clicked.